### PR TITLE
Lobby refactor

### PR DESCRIPTION
**Pull request by nilsonsasaki**
created app_text_divider 
updated TextField to appTextField
updated unnecessary Containers with padding

Divider lines are more accurate with the design
Layout code is more readable
Container creates a Padding widget to implement padding proprieties, removing it  can improve performance

### CODE REVIEW

File: lib/ui/lobby.dart [hunk 1]
```diff
@@ -94,33 +96,12 @@ class _LobbyScreenState extends State<LobbyScreen> {
...
                             child: SizedBox(
-                              height: appBarHeight * 0.75,
+                              height: appBarHeight * 0.8,
                               width: screenWidth * 0.55,
-                              child: TextField(
-                                controller: myController,
-                                keyboardType: TextInputType.number,
-                                inputFormatters: <TextInputFormatter>[
-                                  FilteringTextInputFormatter.digitsOnly
-                                ],
```
Comment (yuumasato) [inline lib/ui/lobby.dart:104]: Can you keep the TextField keyboard with input type number and allowing digitsOnly?

The room ID consists of only numerical digits, and opening the numerical keyboard and allowing only numbers facilitates input.

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: lib/ui/lobby.dart (hunk 1 above)
Comment (nilsonsasaki) [reply lib/ui/lobby.dart]: added keyboardType propriety

Now AppTextField can  change keyboard
updated lobby ApptextField keyboard to numerical keyboard

File: lib/ui/lobby.dart [hunk 2]
```diff
@@ -94,33 +96,13 @@ class _LobbyScreenState extends State<LobbyScreen> {
...
-                              child: TextField(
-                                controller: myController,
+                              child: AppTextField(
+                                textEditingController: myController,
+                                labelText: 'insertRoomCode'.tr,
+                                autofocus: false,
                                 keyboardType: TextInputType.number,
-                                inputFormatters: <TextInputFormatter>[
-                                  FilteringTextInputFormatter.digitsOnly
-                                ],
```
Comment (yuumasato) [inline lib/ui/lobby.dart:104]: And what are your thoughts on keeping the `inputFormatters`?

This prevents any non-digit character from being input into the text field.

**Review by yuumasato — APPROVED** [review]
LGTM, thank you, :)

**Pull request merged**